integrationTest/rgbppTest/testnet/README.md

```diff
@@ -0,0 +1 @@
+execute logs for testnet
```

Comment:
Why is the readme file placed in the `testnet` directory?

Reply:
Empty folders cannot be uploaded, so I added a readme and added a log description.

Reply:
https://www.freecodecamp.org/news/what-is-gitkeep/